qtglean/android.cmake

```diff
@@ -6,6 +6,19 @@ include(${CMAKE_SOURCE_DIR}/scripts/cmake/rustlang.cmake)
 
 set(GLEAN_VENDORED_PATH ${CMAKE_SOURCE_DIR}/3rdparty/glean)
 
+message(STATUS "Building Glean Rust library...")
```

Comment:
What exactly does this bundle build and how does it get consumed by the VPN project?

I have a bit of skepticism around this change, because I don't see how the target architecture would get passed to the resulting artifacts. I would expect this to produce libraries targeting the host architecture when doing a naked `cargo build --releas` like this.